src/frontend/components/property-type/default-type/filter.tsx

```diff
@@ -1,27 +1,65 @@
 import React from 'react'
-import { FormGroup, Input, Select } from '@adminjs/design-system'
+import { Box, FormGroup, Input, Select } from '@adminjs/design-system'
 
 import allowOverride from '../../../hoc/allow-override.js'
 import { FilterPropertyProps } from '../base-property-props.js'
 import PropertyLabel from '../utils/property-label/property-label.js'
 import { useTranslation } from '../../../hooks/use-translation.js'
+import * as BackendFilter from '../../../../backend/utils/filter/filter.js'
+
+const { PARAM_SEPARATOR } = BackendFilter
 
 const Filter: React.FC<FilterPropertyProps> = (props) => {
   const { property, onChange, filter } = props
+
+  const possibleKeys = [
+    property.path,
```

Comment:
keeping contains key as it was before to prevent other adapters to not be able to recognize it